packages/Disclosure/src/disclosure.scss

```diff
@@ -213,14 +213,13 @@
     }
 
     .ids-disclosure__header-content {
-        display: block;
         margin: var(--ids-disclosure-header-content-margin);
+        display: flex;
```

Comment:
this doesn't quite work because the description is supposed to be under the title and it's not doing that:
![image](https://github.com/gsoft-inc/ov-igloo-ui/assets/110498164/e8e08d1d-7c83-4db6-b0e1-08ba6fc0fb5a)
you can see that the description moves up slightly because the top margin is removed.